src/main/java/technology/tabula/TextChunk.java

```diff
@@ -28,11 +29,100 @@ public TextChunk(List<TextElement> textElements) {
     }
 
     // used for "normalizing" RTL languages.
+    // TEMPORARY TODO REMOVE etc.
     public void reverseTextElements(){
         Collections.reverse(this.textElements);
     }
-        
-    
+
+
+    // I hate Java so bad.
+    private HashMap<Byte, String> directionalities = new HashMap<Byte, String>() {{   // BCT = bidirectional character type
```

Comment:
Small stylistic suggestion: `LTR`, `None` and `RTL` could be members of an `enum` instead of literal strings.